ceph/ceph_admin/orch.py

```diff
@@ -19,7 +20,7 @@
 LOG = logging.getLogger()
 
 
-class Orch(LSMixin, PSMixin, RemoveMixin, UpgradeMixin, CephCLI):
+class Orch(LSMixin, PSMixin, RemoveMixin, UpgradeMixin, CephCLI, ApplySpec):
```

Comment:
Let's keep `CephCLI` at the end.

Reply:
As we discussed, apply_spec is part of Orch. so removed ApplySpec from here.